### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by sgarrity** [sdatkinson/NeuralAmpModelerPlugin#209]
When trying to use the standalone NAM app on macOS 13.3.1 (22E261) with a Positive Grid Spark 40 as my audio interface, there's no audio input and unclear behaviour in the Input Device preferences.

**To Reproduce**
1. Install the NAM app (0.7.2)
2. Plug in Positive Grid Spark 40 as audio interface
3. See that there are no input levels
4. Try the Preferences

**Expected behavior**
Expect audio input from the Spark 40, but instead, getting no input and the Preferences show no options for Input 1, and a "1" option for Input 2 but won't actually select it.

**Screenshots**
Here's a quick (51s) screen recording showing the issue with some narration to explain.

https://user-images.githubusercontent.com/58062/232863450-3bc1f9ef-c976-484e-8f00-4639caa713a2.mp4

**Desktop (please complete the following information):**
 - macOS 13.3.1 (22E261) 
 - Plugin version 0.7.2
 - Standalone

Not looking for personal tech support - just hoping to help identify if there's an actual bug. I'm happy to test or provide further info. Thanks.

**Comment by jbrake** [comment]
This is very similar to how it behaves with a Scarlett Solo. I've yet to get it to get the right inputs selected, have never gotten to even try NAM because of this.

**Comment by downcellar-xyz** [comment]
Hey all, I’m using Spark40 on a “dry” setting -> USB-C into M1 MacBook Air -> Logic Pro -> NAM as a Logic track plug-in, and that has worked like a charm for me. I’m happy to test out standalone NAM and try and help debug.

**Comment by sdatkinson** [comment]
I've merged #166, which sums stereo inputs if they're seen.

For folks with Scarlett Solos who want to use the second input, this should be fixed and available in the next release.

Not sure about the Spark--seeing the inputs blank doesn't seem to be the same underlying issue, so leaving this issue open.

**Comment by sgarrity** [comment]
I'm not setup to build/test myself, but when the next release comes out, I'll give it a try and report on any changes in behaviour on the Positive Grid Spark. Thanks!

**Comment by sdatkinson** [comment]
Is this a single-input interface? (I assume yes because it's a guitar amp, but I'm not familiar with it).

Asking because I suppose that #236 may be relevant

**Comment by downcellar-xyz** [comment]
@sdatkinson Yes it is single input, it’s a USB Type B plug out of the back. There are ways to go out of the headphone jack as well but it’s a workaround.

**Comment by sgarrity** [comment]
Yes, the Positive Grid Spark is a single-input, but the output can be stereo.

**Comment by sgarrity** [comment]
Update from version `0.7.3`: Now the "Input 2 (R" menu includes "1" and "2" options (instead of just "1" in the screen-recording from 0.7.2 above. However, the same behaviour occurs: if you try to select either 1 or 2 on the "Input 2 (R" menu, it closes the menu with no selection.

I _was_ able to get 0.7.3 working from within GarageBand though. The above comments are all related to the standalone NAM player.

**Comment by 1BrentEdward** [comment]
This still does not work in stand alone mode for a Yamaha AG06 interface because the guitar input is Input 2 and cannot be changed to input 1.

I can't believe this hasn't been prioritized as there seem to be a lot of interfaces where this doesn't work right on Input 2, and it's been how many months since it was first released?

**Comment by sgarrity** [comment]
An 

> Update from version `0.7.3`: Now the "Input 2 (R" menu includes "1" and "2" options (instead of just "1" in the screen-recording from 0.7.2 above. However, the same behaviour occurs: if you try to select either 1 or 2 on the "Input 2 (R" menu, it closes the menu with no selection.
> 
> I _was_ able to get 0.7.3 working from within GarageBand though. The above comments are all related to the standalone NAM player.

Update here using the latest 0.7.8 release. I'm seeing the same issues described here ⬆️ in the standalone player. Using the plug-in in the context of a DAW is working as intended.
